.github/workflows/test.yml

```
@@ -69,6 +69,9 @@ jobs:
             ]
         include:
            # only test supported ghc major versions
+           - os: ubuntu-latest
+             ghc: '9.4.3'
+             test: true
            - os: ubuntu-latest
              ghc: '9.4.2'
              test: true
```

I think we only run the test suite once per major version, here 9.4.3 would replace 9.4.2. 
```suggestion
           - os: ubuntu-latest
             ghc: '9.4.3'
             test: true
```